script/install_integration_requirements.py

```diff
@@ -0,0 +1,50 @@
+"""Install requirements for a given integration."""
+
+import argparse
+from pathlib import Path
+import subprocess
+
+from .gen_requirements_all import gather_recursive_requirements
+from .scaffold.__main__ import valid_integration
```

Comment:
Side note: `valid_integration` should move to another module. We shouldn't import from a main module.